### PR TITLE
[PXN-2986] Add instrumented tests in CardDrawerView

### PR DESCRIPTION
**Pull request by murijr**
This PR adds instrumented tests to the custom view, as we believe it is the most suitable test for this type of component. We also configure mockk.

### CODE REVIEW

File: carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    @Before
+    fun setUp() {
+        viewScenario.launch(
+            viewFactory = { CardDrawerView(it) },
+            onView = { subject = it }
+        )
+    }
+
+    @Test
+    fun when_is_rendered_with_tag_then_display_tag_with_expected_values() {
```
Comment (samantaalbanez) [inline carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt:40]: what do you think of changing the "_" for " " and adding the`` in the test names?

**Review by cardosodiego — APPROVED** [review]

File: carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+        every { cardDrawerSource.cardUI } returns buildConfiguration(viewScenario.activity)
+        every { cardDrawerSource.tag } returns tag
+        viewScenario.activity.runOnUiThread { subject.show(cardDrawerSource) }
+
+        onView(withText(expectedTagText.toUpperCase())).check(matches(withId(R.id.card_tag)))
+    }
+
+    @Test
+    fun when_is_rendered_with_bottom_label_then_display_tag_with_expected_values() {
```
Comment (luissilva-meli) [inline carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt:54]: remover os "_" nos nomes dos tests

**Review by luissilva-meli — APPROVED** [review]

**Review by samantaalbanez — APPROVED** [review]

File: carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt (hunk 1 above)
Comment (murijr) [reply carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt:40]: For some reason having space or special characters in test function names is not allowed in instrumented tests

File: carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt (hunk 2 above)
Comment (murijr) [reply carddrawer/src/androidTest/java/com/meli/android/carddrawer/model/CardDrawerViewTest.kt:54]: For some reason having space or special characters in test function names is not allowed in instrumented tests.

**Pull request closed without merging**